_minutes/2022-07-21-wecg.md

```diff
@@ -0,0 +1,141 @@
+ * [craig] Google created an autofill API on Android, but on desktop browsers we don't have equivalent capabilities. Users have to juggle enable/disable extensions to avoid conflicts. WebAuthN is a new space that we can't directly integrate with or have to hack into. Android and iOS do a good job of giving the user the opportunity to choose between providers.
+ * [bradley] We're not able to give the user the choice between hardware authenticators or our experience. We need to insert ourselves in between the user and the actual handling.
+ * [simeon] Can anyone speak to the actual WebAuthn RP idea?
+ * [oliver] If you have a hardware authenticator, it needs to store an ID, such as a website. The request is whether the extension can identify itself as the website.
+ * [tim] The ID is automatically the website domain; In Keeper we use postMessage with the web page and communicate with our own native application that sends the request.
+ * [tomislav] How is this different from the [identity.launchWebAuthFlow API](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/identity/launchWebAuthFlow)?
+ * [rob] Given the silence here, it looks like none of the attendees can speak to this topic in detail, so let's ask questions in the issue and continue with the rest of today's meeting topics.
+ * [timothy] Would be helpful to have a spec writeup. Would love to tag in the appropriate folks at Apple.
+
+[Issue 240](https://github.com/w3c/webextensions/issues/240): Support high DPI image assets on the extension store
+
+ * [simeon] Extension store issues are out-of-scope in this CG, but I'm open to hearing from developer concerns right now.
+ * [gaurang] I created [a thread in the chromium-extensions google group](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/obZ9t7b8hPk/m/uRm6_aqTAwAJ), Jackie followed up with discussion of Firefox and Edge's stores. The source images have a high quality but are pixelated after uploading to the CWS.
+ * [simeon] I'm familiar with this problem & actively discussing it with the CWS team.
+
+[Issue 241](https://github.com/w3c/webextensions/issues/241): Event listeners in Content Scripts and communication between main and isolated CS worlds
+
+ * [jack] I don't believe we can do this (breaking change).
+ * [rob] The issue filer is asking to stop firing event listeners in content scripts in response to scripted actions from the web page. This is not a good idea in my opinion; extensions should be prepared to deal with events fired by web pages.
+ * [rob] [event.isTrusted](https://developer.mozilla.org/en-US/docs/Web/API/Event/isTrusted) can already be used if the extension really cares that the event was triggered by the user.
+ * [someone] event.isTrusted is only available to events such as click, but not to CustomEvent.
```

Comment:
@zombie, did you mention this?

Reply:
No, someone else did. I don't recall who.